test/unit/Cardano/Wallet/CoinSelection/FeeSpec.hs

```diff
@@ -313,10 +528,69 @@ propReducedChanges drg (ShowFmt (FeeProp (CoinSelProp utxo txOuts) utxo' (fee, d
     adjust s = left show $ fst $ withDRG drg $ runExceptT $
         adjustForFees feeOpt utxo' s
 
+propFeeEstimation
+    :: FeeProp
+    -> Property
+propFeeEstimation (FeeProp (CoinSelProp utxo txOuts) _ _) = do
+    isRight selection ==> let Right s = selection in prop s
+  where
+    prop coinSel@(CoinSelection inps outs _) = do
+        let (Fee estFee) = estimateCardanoFee cardanoPolicy coinSel
+
+        -- | Make a Hash from a Base16 encoded string, without error handling.
+        let hash16 :: ByteString -> Hash a
+            hash16 = either bomb Hash . convertFromBase Base16
+                where
+                    bomb msg = error ("Could not decode test string: " <> msg)
+
+        -- | Make an Address from a Base58 encoded string, without error handling.
+        let addr58 :: ByteString -> Address
+            addr58 = maybe (error "addr58: Could not decode") Address
+                . decodeBase58 bitcoinAlphabet
+
+        let inputId0 = hash16 "60dbb2679ee920540c18195a3d92ee9be50aee6ed5f891d92d51db8a76b02cd2"
+        let address0 = addr58 "DdzFFzCqrhsug8jKBMV5Cr94hKY4DrbJtkUpqptoGEkovR2QSkcA\
+                              \cRgjnUyegE689qBX6b2kyxyNvCL6mfqiarzRB9TRq8zwJphR31pr"
```

Comment:
This _looks like_ an address with a derivation path payload (a.k.a an address from the random derivation scheme); Those are bigger in practice than the one we would use. It'd be nice to actually have the fee calculation depends on the address scheme we use, so that we can adjust fee depending on this :)

Reply:
indeed, the examples were for Testnet/Random scheme